.github/workflows/validate-pr.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: Missing `area/` label
         if: contains(join(github.event.pull_request.labels.*.name, ','), 'impact/') && !contains(join(github.event.pull_request.labels.*.name, ','), 'area/')
         run: |
-          echo "Every PR with an \`impact/*\` label should also have an \`area/*\` label"
+          echo "::error::Every PR with an 'impact/*' label should also have an 'area/*' label"
           exit 1
```

Comment:
As follow-up we could look at using https://github.com/actions/github-script:

```yaml
uses: actions/github-script@v7
with:
  script: |
    core.setFailed(`Every PR with an 'impact/*' label should also have an 'area/*' label`);
```

Reply:
`core` module API here: https://github.com/actions/toolkit/tree/main/packages/core#actionscore

Reply:
Is there any visual difference between these though?